### ISSUE TITLE
Event ID not exist issue occur when running websocket

### ISSUE THREAD

**Issue by qiaojingx** [01org/zephyr.js#1150]
### Description

Event ID not exist issue occur when running websocket

### Test Code
[WebSocketServer.js](https://github.com/01org/zephyr.js/blob/master/samples/websockets/WebSocketServer.js);

### Steps to Reproduction

1. `make JS=samples/websockets/WebSocketServer.js`
2. `make dfu`
3. connect via BLE then add the IP route for the Arduino's 6lowpan address
   ` ip -6 route add 2001:db8::/64 dev bt0`
3. Execute `node websockets/NodeWebSocketClient.js` on Linux

### Actual Result

callback_id not found issue occurs.
![selection_002](https://cloud.githubusercontent.com/assets/19813546/26050847/d38be8d0-3992-11e7-98ca-f91a246c9303.png)
![selection_001](https://cloud.githubusercontent.com/assets/19813546/26050848/d38c78d6-3992-11e7-89ad-522fa2420d7f.png)

### Expected Result

Connection setup.
![selection_004](https://cloud.githubusercontent.com/assets/19813546/26050864/e2bf5d1e-3992-11e7-9059-8113275a9e4b.png)

### Test Builds

Branch | Commit Id | Target Device | Test Date | Result
---|---|---|---|---
master | bd07595 | Arduino 101 | May 15, 2017 | Pass
master | bd07595 | QEMU | May 15, 2017 | Pass
master | e5f62c0 - e24821c| Arduino 101 | May 15, 2017 | Fail
master | e5f62c0 - e24821c| QEMU | May 15, 2017 | Fail

### Additional Information

This issue is probably caused by commit e5f62c0

**Comment by qiaojingx** [comment]
Test with latest commit c2ca034 on Arduino101 and QEMU.
This issue is still reproducible on QEMU, but different on Arduino101.  Websocket connection doesn't setup and no error print.
Arduino 101:
![selection_091](https://user-images.githubusercontent.com/19813546/27727484-3deca380-5db1-11e7-8d17-937a69411745.png)
QEMU:
![selection_093](https://user-images.githubusercontent.com/19813546/27727485-3e1b57ac-5db1-11e7-993d-27d16b2a955f.png)

**Comment by qiaojingx** [comment]
Verified with commit de91ade + #1374. This issue is not reproduced on QEMU. But websocket connection is blocked by #1348 on Arduino 101.
I will verify this issue on arduino later, after #1348 is fixed.

**Comment by qiaojingx** [comment]
Verified with commit 5c65f11 on Arduino 101, This issue is not reproduced when first connect. But got new issue: #1386 . Another similar issue occurs when try to re-connect, tracking by #1385.
Close this one.
